e107_themes/landingzero/languages/English.php

```diff
@@ -21,7 +21,7 @@
 define("LAN_LZ_THEME_11", "Phone");
 define("LAN_LZ_THEME_12", "Your message here...");
 define("LAN_LZ_THEME_13", "Get in Touch");
-define("LAN_LZ_THEME_14", "We love feedback. Fill out the form below and we\'ll get back to you as soon as possible.");
+define("LAN_LZ_THEME_14", "We love feedback. Fill out the form below and we&apos;ll get back to you as soon as possible.");
```

Comment:
Thanks. Did you try replacing `we\'ll ` with `we'll`

Reply:
I did and it did not display properly... What I can tell you is the '\' shows in the rendered text and is wrong ;-)

Reply:
that was supposed to be the backslash char in the ''